src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2867,6 +2867,6 @@ void M1()
             compilation.VerifyDiagnostics();
         }
 
-#endregion
+        #endregion
```

Comment:
nit: I expected `#region` and `#endregion` to get left-aligned in our code style. Is that not the case?

Reply:
No. Region directives are always indented; there is no internal or public option to change this behavior.

Reply:
Should we log a bug to allow left-aligned region directives?

Reply:
@cston You can file a feature request, but keep in mind it would likely have the same result as #12306 considering that https://github.com/dotnet/roslyn/issues/12306#issuecomment-412672491 applies to this.

Reply:
Hmm, I surprised to see that the `#region` directives in Roslyn are all indented. Perhaps that is our code style after all.

---
In reply to: [306585972](https://github.com/dotnet/roslyn/pull/37433#discussion_r306585972) [](ancestors = 306585972)